classes/Carrier.php

```diff
@@ -295,14 +295,14 @@ public function getDeliveryPriceByWeight($total_weight, $id_zone)
         $id_carrier = (int) $this->id;
         $cache_key = $id_carrier . '_' . $total_weight . '_' . $id_zone;
         if (!isset(self::$price_by_weight[$cache_key])) {
-            $sql = 'SELECT d.`price`
-					FROM `' . _DB_PREFIX_ . 'delivery` d
-					LEFT JOIN `' . _DB_PREFIX_ . 'range_weight` w ON (d.`id_range_weight` = w.`id_range_weight`)
-					WHERE d.`id_zone` = ' . (int) $id_zone . '
-						AND ' . (float) $total_weight . ' >= w.`delimiter1`
-						AND ' . (float) $total_weight . ' < w.`delimiter2`
-						AND d.`id_carrier` = ' . $id_carrier . '
-						' . Carrier::sqlDeliveryRangeShop('range_weight') . '
+            $sql = 'SELECT d.`price`
```

Comment:
I don't understand the modifications for this file? Seems like spaces were turned into tabs Is it some mistake added from your IDE or something?
Anyway I think you should revert those modifications, we don't use tab for indentation

Reply:
@jolelievre & @matthieu-rolland Affirmative : it was a lot of tabs before. So it's better in spaces.